### ISSUE TITLE
Config and training loss curves

### ISSUE THREAD

**Issue by KuzmaKhrabrov** [MinkaiXu/AliDiff#3]
Could you, please, help with the following:

In training_ipo.yml, max_grad_norm=1000, dpo_beta=1000 and lr=2.048e-6 ; while in the paper the values max_grad_norm=8, beta=5 and lr=5e-6 are given in the training details. What were the parameters for the models, which metrics are listed in the paper?

Additionally, have you saved the training loss curves or final metrics of the models? In my attempts to reproduce training, train/val losses practically did not  change during the process.

**Comment by DWF09** [comment]
> Could you, please, help with the following:
> 
> In training_ipo.yml, max_grad_norm=1000, dpo_beta=1000 and lr=2.048e-6 ; while in the paper the values max_grad_norm=8, beta=5 and lr=5e-6 are given in the training details. What were the parameters for the models, which metrics are listed in the paper?
> 
> Additionally, have you saved the training loss curves or final metrics of the models? In my attempts to reproduce training, train/val losses practically did not change during the process.

Hello, may I ask how you obtained the author's pre-trained model?

**Comment by KuzmaKhrabrov** [comment]
> > Could you, please, help with the following:
> > In training_ipo.yml, max_grad_norm=1000, dpo_beta=1000 and lr=2.048e-6 ; while in the paper the values max_grad_norm=8, beta=5 and lr=5e-6 are given in the training details. What were the parameters for the models, which metrics are listed in the paper?
> > Additionally, have you saved the training loss curves or final metrics of the models? In my attempts to reproduce training, train/val losses practically did not change during the process.
> 
> Hello, may I ask how you obtained the author's pre-trained model?

I tried to repeat the pre-training myself, starting from the ipdiff pre-trained model.

**Comment by ShuoAndy** [comment]
> > > Could you, please, help with the following:
> > > In training_ipo.yml, max_grad_norm=1000, dpo_beta=1000 and lr=2.048e-6 ; while in the paper the values max_grad_norm=8, beta=5 and lr=5e-6 are given in the training details. What were the parameters for the models, which metrics are listed in the paper?
> > > Additionally, have you saved the training loss curves or final metrics of the models? In my attempts to reproduce training, train/val losses practically did not change during the process.
> > 
> > 
> > Hello, may I ask how you obtained the author's pre-trained model?
> 
> I tried to repeat the pre-training myself, starting from the ipdiff pre-trained model.

May I ask whether you were able to successfully finish the training? If so, did you save any checkpoints that you would be willing to share?

**Comment by ShuoAndy** [comment]
> > > Could you, please, help with the following:
> > > In training_ipo.yml, max_grad_norm=1000, dpo_beta=1000 and lr=2.048e-6 ; while in the paper the values max_grad_norm=8, beta=5 and lr=5e-6 are given in the training details. What were the parameters for the models, which metrics are listed in the paper?
> > > Additionally, have you saved the training loss curves or final metrics of the models? In my attempts to reproduce training, train/val losses practically did not change during the process.
> > 
> > 
> > Hello, may I ask how you obtained the author's pre-trained model?
> 
> I tried to repeat the pre-training myself, starting from the ipdiff pre-trained model.

I trained the model using:
`python scripts/train_ipo.py configs/training_ipo.yml`

The training itself finishes without errors and checkpoints are saved normally. However, when I try to run sampling with the resulting checkpoints, the diffusion trajectories completely collapse (e.g., highly unstable or nonsensical ligand trajectories), making the checkpoints unusable for generation.

**Comment by ShuoAndy** [comment]
I have completed the training following the settings in training_ipo.yml (max_grad_norm=1000, dpo_beta=1000, and lr=2.048e-6). Although the train/val losses practically did not decrease during the training process, the model's performance in terms of binding affinity has indeed improved as reflected in the metrics reported in the paper.
